Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM quay.io/centos/centos:stream8
+FROM quay.io/centos/centos:stream9
 
-RUN dnf update -y && \
+RUN dnf upgrade -y && \
```

Comment:
(Thinking aloud) we should probably remove the whole `RUN`, we don't use any packages in this container.